scripts/output/rds_report.R

```diff
@@ -26,20 +26,20 @@ if(!exists("source_include")) {
 }
 
 cfg <- gms::loadConfig(file.path(outputdir, "config.yml"))
-gdx	<- file.path(outputdir,"fulldata.gdx")
+gdx	<- file.path(outputdir, "fulldata.gdx")
 rds <- paste0(outputdir, "/report.rds")
 mif <- paste0(outputdir, "/report.mif")
-runstatistics <- paste0(outputdir,"/runstatistics.rda")
+runstatistics  <- paste0(outputdir, "/runstatistics.rda")
 resultsarchive <- "/p/projects/rd3mod/models/results/magpie"
 ###############################################################################
 
 
-report <- getReport(gdx,scenario = cfg$title)
-write.report2(report, file=mif)
+report <- getReport(gdx, scenario = cfg$title, dir = outputdir)
+write.report2(report, file = mif)
```

Comment:
```suggestion
write.report(report, file = mif)
```
not directly related to your changes, but `write.report2` had recently been deprecated and is now just a redirect to `write.report`. So using the latter function directly is recommended

Reply:
Done. Also updated all other instances where this occurs.